UnitsNet/BaseDimensions.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) 2013 Andreas Gullberg Larsen (andreas.larsen84@gmail.com).
+// https://github.com/angularsen/UnitsNet
+// 
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+// 
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+// 
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+using System;
+
+namespace UnitsNet
+{
+    public struct BaseDimensions
+    {
+        public readonly int
```

Comment:
You should add `/// <summary></summary>` to each of these.
I think it would also be better to use the unit name and not the measure name: Meter, Kilogram, Second, Ampere, Kelvin, Mole, Candela (https://en.wikipedia.org/wiki/SI_base_unit)

I would personally prefer to use "Meter" for consistency in the lib, but I see that I might be colored by coming from a germanic language and according to some googling "Metre" is the preferred spelling almost everywhere outside US and a couple of scandinavian countries. I still argue for consistency and rather changing all spelling in one go in a major version bump.